cfdtools/data.py

```diff
@@ -150,8 +156,12 @@ def dataSet_spectrogram(self, datafilter=None, window=None):
 
 
 class DataSetList(DataSetBase):
+
+    _version = 1
     _available_Xrep = ('nodal', 'cellaverage', 'spectralcell')
     _available_Trep = ('instant', 'timeevol', 'pod')
+    # defines the names of data items that should be written as attributes; not hdf5 dataset
+    _properties = ('time',)
```

Comment:
This name `_properties` is already used in the parent class.

Reply:
bien vu